conan-recipes/from-source/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "0.10.1-rc":
+    url: "https://github.com/eclipse-zenoh/zenoh-c/archive/refs/tags/0.10.1-rc.tar.gz"
+    sha256: "9fb703eff79b7c1ef216f411dadcb7f0132c8ba665823b5a8252b467b4937438"
```

Comment:
I'm not very experienced with conan packaging, so forgive me if the answer to this is obvious: Is there a way to build whatever is currently checked out in the git repo? There may be times where we want to test downstream projects using a specific commit of zenoh-c.

Reply:
@oteffahi That makes sense. Let's keep this recipe as is - better to have something that is correct and in line with what CCI would want, even this can't be submitted there _yet_. If we do ever find a need to build a development package, we'll figure out how to do it then.

Feel free to mark this comment resolved.

Reply:
@gregmedd We've experimented with compiling specific commits, and it seems pretty straight-forward to add. As far as I know this not in-line with CCI's recipe requirements, but since we're managing our own recipe we might as well add exclusive features that cannot be supported otherwise 😄